### PR TITLE
Fix for WaitForWork in Preemphasis

### PR DESCRIPTION
**Pull request by szalpal**
Signed-off-by: Michał Szołucha <mszolucha@nvidia.com>

#### Why we need this PR?
There was no WaitForWork after DoWorkWithID, this PR fixes it.

### CODE REVIEW

File: dali/test/python/test_operator_preemph.py [hunk 1]
```diff
@@ -65,5 +65,6 @@ def perform(preemph_coeff):
         assert isclose(a, b, abs_tol=eps)
 
 
-perform(0.)
-perform(0.5)
+def test_preemphasis():
+    body(0.0)
```
Comment (jantonguirao) [inline dali/test/python/test_operator_preemph.py]:
```suggestion
   for coef in [0.5, 0.0]:
       yield check_preemphasis_operator, coef
```
I'd also consider adding the shape as an argument

File: dali/test/python/test_operator_preemph.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def iter_setup(self):
         self.feed_input(self.data, premade_batch)
 
 
-def perform(preemph_coeff):
+def body(preemph_coeff):
```
Comment (jantonguirao) [inline dali/test/python/test_operator_preemph.py]:
```suggestion
def check_preemphasis_operator(preemph_coeff):
```

File: dali/test/python/test_operator_preemph.py [hunk 3]
```diff
@@ -65,5 +65,6 @@ def perform(preemph_coeff):
         assert isclose(a, b, abs_tol=eps)
 
 
-perform(0.)
-perform(0.5)
+def test_preemphasis():
```
Comment (jantonguirao) [inline dali/test/python/test_operator_preemph.py]:
```suggestion
def test_preemphasis_operator():
```

**Comment by szalpal** [comment]
!build

**Pull request merged**